### DISCUSSION TITLE
Does anyone know what style of artwork/image these are considered?

### DISCUSSION THREAD

**Discussion by iTheMaverick** [community/community#76699] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/141296807/de77ccee-4469-41b0-b41b-ce55166101dd)
![image](https://github.com/community/community/assets/141296807/876880de-d7c0-42db-b085-b9325a1591be)

**Comment by MantasTech** [comment]
Good day [iTheMaverick](https://github.com/iTheMaverick),

Theese are minimalist and contemporary style images. It's a common style for web and application graphics.
